src/construction.cpp

```diff
@@ -331,23 +331,26 @@ void construction_menu()
             int available_buffer_height = w_height - 7 - 3;
             int available_window_width = w_width - ( w_list_width + w_list_x0 + 2 ) - 1;
             nc_color color_stage = c_white;
-
+            // print the hotkeys regardless of if there are constructions
+            trim_and_print( w_con, w_height - 5, ( w_list_width + w_list_x0 + 2 ),
+                       available_window_width, c_white,
+                       _("Press %s or %s to tab."), 
+                       ctxt.get_desc("LEFT").c_str(),
+                       ctxt.get_desc("RIGHT").c_str() );
+            mvwprintz( w_con, w_height - 4, ( w_list_width + w_list_x0 + 2 ), c_white,
+                       _( "Press %s to search." ), ctxt.get_desc( "FILTER" ).c_str() );
+            // Print instructions for toggling recipe hiding.
```

Comment:
That's not mine actually, it was where this general section of code used to live.

I'll get rid of it anyway.